build-tools/packages/build-cli/package.json

```diff
@@ -171,7 +171,7 @@
 		"eslint-config-oclif": "^5.2.1",
 		"eslint-config-oclif-typescript": "^3.1.12",
 		"eslint-config-prettier": "~9.1.0",
-		"jssm-viz-cli": "5.101.0",
+		"jssm-viz-cli": "^5.101.0",
```

Comment:
Not 5.104.1?

Reply:
No. [jssm-viz-cli](https://www.npmjs.com/package/jssm-viz-cli?activeTab=versions) hasn't had a new release; it's the transitive dep it has on jssm-viz that is really being bumped.